From 32920c9e8c4044012391b07dd6721adf9e488996 Mon Sep 17 00:00:00 2001
From: Jakub Ruzicka <jruzicka@redhat.com>
Date: Thu, 12 Sep 2013 16:04:19 +0200
Subject: [PATCH] Remove runtime dependency on python-pbr

---
 cinderclient/__init__.py | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/cinderclient/__init__.py b/cinderclient/__init__.py
index 29e35c8..bf2e5a8 100644
--- a/cinderclient/__init__.py
+++ b/cinderclient/__init__.py
@@ -14,12 +14,4 @@
 
 __all__ = ['__version__']
 
-import pbr.version
-
-version_info = pbr.version.VersionInfo('python-cinderclient')
-# We have a circular import problem when we first run python setup.py sdist
-# It's harmless, so deflect it.
-try:
-    __version__ = version_info.version_string()
-except AttributeError:
-    __version__ = None
+__version__ = "REDHATCINDERCLIENTVERSION"
