From a0d236fcdeffe71aeac201c3fdb90785b63276ab Mon Sep 17 00:00:00 2001
From: Jakub Ruzicka <jruzicka@redhat.com>
Date: Thu, 12 Sep 2013 16:14:40 +0200
Subject: [PATCH] Stop pbr from installing requirements during build

---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index 70c2b3f..afab729 100644
--- a/setup.py
+++ b/setup.py
@@ -18,5 +18,4 @@
 import setuptools
 
 setuptools.setup(
-    setup_requires=['pbr'],
     pbr=True)
